code/game/jobs/job/command/auxiliary/crew_chief.dm

```diff
@@ -1,19 +1,19 @@
 /datum/job/command/crew_chief
-	title = JOB_DROPSHIP_CREW_CHIEF
-	total_positions = 2
-	spawn_positions = 2
+	title = JOB_DROPSHIP_PILOT
+	total_positions = 1
+	spawn_positions = 1
```

Comment:
```suggestion
	total_positions = 2
	spawn_positions = 2
```
Keep these two. Let there be a co-pilot that can tag team and support the other.

Reply:
Sure, I don't see why not!